packages/react-admin/src/mui/auth/Login.js

```diff
@@ -99,23 +65,8 @@ const sanitizeRestProps = ({
  *     );
  */
 class Login extends Component {
```

Comment:
You should turn it into a functional component since it has no other method than `render()`